R/HTTPError.R

```diff
@@ -5,7 +5,7 @@
 #'
 #' @description
 #' Helps to generate http error responses See [raise] for example.
-#' Contains [HTTPErrorFactory] class for the exception in the user's code.
+#' Contains `HTTPErrorFactory` class for the exception in the user's code.
```

Comment:
I would call `RestRserve:::HTTPErrorFactory` because is not available from the docs index

Reply:
`[HTTPErrorFactory] was broken link.

Reply:
Yeah, I got it. Let's put `RestRserve:::HTTPErrorFactory`